modules/migrations/base/downloader.go

```diff
@@ -20,7 +27,8 @@ type Downloader interface {
 	GetReleases() ([]*Release, error)
 	GetLabels() ([]*Label, error)
 	GetIssues(page, perPage int) ([]*Issue, bool, error)
-	GetComments(issueNumber int64) ([]*Comment, error)
+	GetComments(opts GetCommentOptions) ([]*Comment, bool, error)
+	SupportGetRepoComments() bool
```

Comment:
This is kinda at odds with the rest of this interface whereby if something is not supported it usually returns an ErrNotSupported.

It would probably be more consistent to provide a GetAllComments function instead that most migrators would ErrNotSupport and if so we fallback to GetComments(issueNumber) and GetAllComments which Github would support that would mean we could do the all comments route instead. Otherwise this is kinda inconsistent.

Reply:
The `GetAllComments` have to invoke after `GetIssues` and `GetPullRequests`. But `GetComment` should be invoked after every issue or pull request. So we cannot try `GetAllComments` at first.